packages/@aws-cdk/aws-s3/lib/bucket.ts

```diff
@@ -434,6 +435,9 @@ export interface BucketAttributes {
    * regions launched since 2014.
    *
    * @default false
+   *
+   * Not needed to specify anymore - proper website url format is derived from the bucket region.
```

Comment:
Any reason you dropped the `@deprecated` flag here?

Reply:
I removed it, because when I wanted to test `bucketWebsiteNewUrlFormat:` explicitly in the test `test('old bucketWebsiteUrl format with explicit bucketWebsiteNewUrlFormat'` but I got a DeprecationException when running the test, was not able workaround this exception :/ If it's possible I will fix it, just tell me how please 😁
I think it must be possible, because deprecated stuff needs to be tested usually.

Reply:
@fynnfluegge Took me a while as well, but there's a helper to test deprecated feature:
```ts
import { testDeprecated } from '@aws-cdk/cdk-build-tools';
```

I took the liberty to add the deprecation, change the tests and fix some other documentation texts.

Reply:
Thank you, Perfect! 👌